Detections/MultipleDataSources/SuspiciousAWSEC2ComputeResourceDeployments.yaml

```diff
@@ -0,0 +1,54 @@
+id: 9e457dc4-81f0-4d25-bc37-a5fa4a17946a

```

Comment:
Given this just uses AWS data it should go in the AWS solution folder rather than in the multiple data sources detection one

Reply:
The file was moved to the AWS solution folder.